AvalonStudio/AvalonStudio.Extensibility/ExtensionManifest.cs

```diff
@@ -7,47 +7,64 @@ namespace AvalonStudio.Extensibility
 {
     internal class ExtensionManifest : IExtensionManifest
     {
-        private const string MefComponentsString = "mefComponents";
+        private static string DefaultIcon = "resm:AvalonStudio.Assets.logo-256.png?assembly=AvalonStudio";
+        private static JsonSerializer Serializer = new JsonSerializer();
```

Comment:
you could use this api instead...
https://github.com/VitalElement/AvalonStudio/blob/master/AvalonStudio/AvalonStudio.Extensibility/Utils/SerializedObject.cs#L7

Reply:
example here:
https://github.com/VitalElement/AvalonStudio/blob/ac50a893d87512f55ac19bf437b519aebca39132/AvalonStudio/AvalonStudio.Controls.Standard/WelcomeScreen/RecentProjectsCollection.cs#L49

https://github.com/VitalElement/AvalonStudio/blob/ac50a893d87512f55ac19bf437b519aebca39132/AvalonStudio/AvalonStudio.Controls.Standard/WelcomeScreen/RecentProjectsCollection.cs#L35

This used NewtonSoft.JSON.

Reply:
Deserializing is not the best option in this case, as the object needs to be initialized before with the manifest path. Maybe I should add a `PopulateObject` method to `SerializedObject`?

Reply:
Could do, would be nice to share Json serialization code so we don't have several instances.